### DISCUSSION TITLE
On click Link > URL property will redirect to error page [Bug]

### DISCUSSION THREAD

**Discussion by iroshandezilva** [webstudio-is/webstudio-community#40] [Q&A]
Please view below screen recording to find the mentioned bug

https://github.com/webstudio-is/webstudio/assets/55875936/e7052ddf-0d26-4e8f-a1b2-85a109846ca3

**Comment by kof** [comment]
Looks like a second click on the already active tab throws this error, thanks for the report!

**Comment by kof** [comment]
Can reproduce in prod, but can't reproduce locally, seems already fixed, will be deployed today most likely

**Comment by kof** [comment]
Fixed, please reopen if you still see it.
